docs/cli/getting-started.md

```diff
@@ -67,7 +68,17 @@ The API driven plugin discovery feature is enabled as default method to install
   ```sh
   tanzu plugin list
   ```
+###### Option 2: Using install script
```

Comment:
Same - need a blank line after the title line to make the linter happy.

Reply:
Added the blank lines and checked with doc-linter. Seems green